brew_view/base_handler.py

```diff
@@ -127,12 +128,18 @@ def write_error(self, status_code, **kwargs):
                 code = error_dict.get('status_code', 500)
                 message = error_dict.get('message', str(e))
 
+            # HTTPError is a special case - can be raised by Tornado itself
+            elif isinstance(e, HTTPError):
```

Comment:
Are we capable of raising a new error now? It seems like maybe we were just missing this case before?

Reply:
Oh yeah, I'll pull this back out. I kind of found it by accident. I think we should still do this, but it doesn't need to be in this merge.